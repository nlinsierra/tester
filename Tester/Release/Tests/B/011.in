9C 4C JD KH 2S JC 5S 2C AS TS TC AC 8H 7D 8S 2D 3D AD 8D 4S QD
