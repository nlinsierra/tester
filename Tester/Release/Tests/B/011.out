31
3C 5C 6C 7C 8C QC KC 4D 5D 6D 9D TD KD 2H 3H 4H 5H 6H 7H 9H TH JH QH AH 3S 6S 7S 9S JS QS KS
